
Header header
uint8[] lut
ColorSpec[] colors
